### PR TITLE
Feature/angular svelte style

### PR DESCRIPTION
**Pull request by decadef20**
## Description

Add a short description of:
- what changes you made, 
#544 
- why you made them, and 
- any other context that you think might be helpful for someone to better understand what is contained in this pull request. 

This sort of information is useful for people reviewing the code, as well as anyone from the future trying to understand why changes were made or why a bug started happening.

### CODE REVIEW

**Comment by decadef20** [comment]
It's so weird. All tests pass on my machine and I pushed. I don't know why CI failed.  Any suggestions here? @samijaber

**Comment by samijaber** [comment]
@decadef20 You might need to run `yarn` to get the newest version of the svelte `prettier` plugin

File: packages/core/src/generators/svelte.ts [hunk 1]
```diff
@@ -299,6 +309,19 @@ const FUNCTION_HACK_PLUGIN: Plugin = () => ({
...
+    const child = children[i];
+    if (child.bindings.style || child.properties.style) {
+      return true;
+    } else if (child.children.length) {
+      return hasStyleObject(child.children);
+    }
+  }
+
+  return false;
+};
```
Comment (samijaber) [inline packages/core/src/generators/svelte.ts:323]: We use `traverse` for this type of logic in our codebase. See this example:

https://github.com/BuilderIO/mitosis/blob/9fc4c5d26eb02ba9c933ee9726eada8699b1659d/packages/core/src/helpers/styles/helpers.ts#L16-L29

It's simpler and avoids the need for correctly performing recursion.

Comment (samijaber) [reply packages/core/src/generators/svelte.ts:323]: NOTE: The function I'm sharing is checking for node.bindings.css, so you can't reuse it, but you can write a similar one with a different check

**Review by samijaber — APPROVED** [review]
Lovely!

File: packages/core/src/generators/svelte.ts (hunk 1 above)
Comment (decadef20) [reply packages/core/src/generators/svelte.ts:323]: Okay

**Pull request merged**